src/transformers/models/gpt_neox_japanese/modeling_gpt_neox_japanese.py

```diff
@@ -273,9 +380,29 @@ def rotate_half(x):
     return torch.cat((-x2, x1), dim=-1)
 
 
-def apply_rotary_pos_emb(q, k, cos, sin, offset: int = 0):
```

Comment:
this is potentially breaking no? (no more offset)

Reply:
Hmm right, lemme check this

Reply:
update: just verified we don't need to slice anymore, because we apply rope directly on the curretn position. Prev we applied Rope for all positions up to the current and had to slice out cached positions